src/gui/gui.pri

```diff
@@ -79,6 +78,28 @@ win32|macx {
     SOURCES += $$PWD/programupdater.cpp
 }
 
+# notifications
+HEADERS += \
+    $$PWD/notifications/notifier.h
+
+SOURCES += \
+    $$PWD/notifications/notifier.cpp
+
+unix:!macx:dbus {
```

Comment:
I'm not sure. DBus can be installed on Mac, but desktop notifications will not work, AFAIK.